docs/source/dev-setup/devenv.rst

```diff
@@ -36,6 +36,36 @@ the application after installing it:
 
     open /Applications/Docker.app
 
+Developing on Windows
+~~~~~~~~~~~~~~~~~~~~~
+
+On Windows 10 you should use the native Docker distribution and you
+may use the Windows PowerShell. However, for the ``binaries``
+command to succeed you will still need to have the ``uname`` command
+available. You can get it as part of Git but beware that only the
+64bit version is supported.
+
+Before running any ``git clone`` commands, run the following commands:
+
+::
+
+    git config --global core.autocrlf false
+    git config --global core.longpaths true
```

Comment:
Changing the global config means it will impact all repositories. For the purposes of this tutorial, it probably makes more sense to do a bare clone followed by local config updates before checking out the source. Something like:

```
git clone --bare https://github.com/hyperledger/fabric
git config --local core.autocrlf false
git config --loal core.longpaths true
git checkout master
```